### PR TITLE
azurerm_nginx_deployment - changing sku now creates a new resource

### PR DESCRIPTION
**Pull request by puneetsarna**
The NGINXaaS API does not support changing SKUs.

### CODE REVIEW

**Comment by puneetsarna** [comment]
@agazeley

**Comment by agazeley** [comment]
LGTM

File: internal/services/nginx/nginx_deployment_resource.go [hunk 1]
```diff
@@ -76,6 +76,7 @@ func (m DeploymentResource) Arguments() map[string]*pluginsdk.Schema {
 			// docs: <https://docs.nginx.com/nginx-for-azure/billing/overview/>
 			Type:     pluginsdk.TypeString,
 			Required: true,
+			ForceNew: true,
```
Comment (katbyte) [inline internal/services/nginx/nginx_deployment_resource.go:79]: Could we update the documentation to reflect this?

**Review by katbyte — CHANGES_REQUESTED** [review]

File: internal/services/nginx/nginx_deployment_resource.go (hunk 1 above)
Comment (puneetsarna) [reply internal/services/nginx/nginx_deployment_resource.go:79]: Thanks @katbyte . I have updated the documentation to call this out 👍

**Review by katbyte — APPROVED** [review]
Thankis @puneetsarna ! LGTM ☎️

**Pull request merged**